app/src/main/java/org/phenoapps/verify/MainActivity.java

```diff
@@ -552,9 +554,6 @@ private void loadBarcodes() {
 
     private synchronized void askUserExportFileName() {
 
-        AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
Is there a reason you removed this dialog? Right now the default file name is confusing, I think it's useful for the user to choose it before the intent starts.